pages/about/groups/task-forces/conformance-testing.md

```diff
@@ -32,86 +30,63 @@ Some information on this page is also shown on, and may be more current in, the
-- GitHub repository: [https://github.com/w3c/wcag-act-rules/](https://github.com/w3c/wcag-act-rules/)
+* [Minutes from previous meetings](https://www.w3.org/WAI/GL/task-forces/conformance-testing/minutes)
+* [Mailing list archive](http://lists.w3.org/Archives/Public/public-wcag-act/)
 
-## How to Comment, Contribute, and Participate
+## Publications and copyright
 
-To join the ACT TF, individuals must be participants of the AG WG. Participants are expected to [actively contribute](/about/groups/task-forces/conformance-testing/work-statement/#participation) to the work of the Task Force. If you are interested in participating in the ACT TF, please send e-mail to: [Daniel Montalvo](mailto:dmontalvo@w3.org?subject=ACT%20TF%20Enquiry) and include a little bit about what you're interested in and how you think that you may be able to contribute to the Task Force. Then follow the [AG Working Group participation](/about/groups/agwg/participate/) procedures to join the Working Group, and once you have joined ask [Daniel Montalvo](mailto:dmontalvo@w3.org) to add you to the task force.
+The ACT Task Force has published the following resources under the name of the AG Working Group:
 
-To contribute without joining the task force, see the [AG Working Group contribute page](/about/groups/agwg/#how-to-comment-contribute-and-participate) for general instructions. To contribute to documents under development, see [GitHub repository](https://github.com/w3c/wcag-act/) directly.
+* [Accessibility Conformance Testing (ACT) Rules Format 1.0 — W3C Recommendation](https://www.w3.org/TR/act-rules-format/)
+* [Accessibility Conformance Testing (ACT) Rules Format 1.1 — W3C First Public Working Draft](https://www.w3.org/TR/act-rules-format-1.1/)
+* [ Accessibility Conformance Testing (ACT) Rules Format 1.1 — W3C Editor’s Draft](https://w3c.github.io/wcag-act/act-rules-format.html)
+* [Repository of published Accessibility Conformance Testing (ACT) Rules](https://www.w3.org/WAI/standards-guidelines/act/rules/)
 
-[Current participants in the ACT TF](https://www.w3.org/2000/09/dbwg/details?group=93339&public=1).
+W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group — see the list and instructions for disclosing a patent on [Specifications published by the Group — Intellectual property rights — AG Working Group](https://www.w3.org/groups/wg/ag/ipr/#specs).
 
-### Work Statement
+## Contact the task force
 
-The [Accessibility Conformance Testing (ACT) Task Force Work Statement](/about/groups/task-forces/conformance-testing/work-statement/) defines the initial objective, scope, approach, and participation of the Task Force.
\ No newline at end of file
+If you have a question for the task force, email the [chairs](https://www.w3.org/groups/tf/wcag-act/participants/#chairs) or the [W3C staff contact](https://www.w3.org/groups/tf/wcag-act/participants/#staff). 
```

Comment:
Hmm... irritating. No chairs/TC group email. 

Ah well... only thing to change is the heading then:

> ## Contact the chairs